more/free-programming-interactive-tutorials-en.md

```diff
@@ -161,6 +161,7 @@
 * [Go Tutorial](https://www.w3schools.com/go/) - W3Schools
 * [Learn Go](https://www.codecademy.com/learn/learn-go) - Google, Robert Griesemer, Rob Pike, and Ken Thompson (Codecademy) *(account required)*
 * [Start using Go](https://docs.microsoft.com/en-us/learn/paths/go-first-steps/) - Microsoft
+* [TCP Servers in Go](https://app.codecrafters.io/concepts/go-tcp-server) - CodeCrafters *(no account required)*
```

Comment:
```suggestion
* [TCP Servers in Go](https://app.codecrafters.io/concepts/go-tcp-server) - CodeCrafters
```
Same issue with the other 2

Reply:
@eshellman I have removed the required texts. Hope this PR looks good now.